product/stream_processor/integrations/events/event_handler.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any
+
+from product.models.products.product import ProductChangeNotification
+from product.stream_processor.integrations.events.base import EventHandler, EventProvider
+from product.stream_processor.integrations.events.functions import build_events_from_models
```

Comment:
how come this function is not a private method of the handler? or maybe in in the interface they extend? its super generic right?

Reply:
yeah, it is now after all the work, it wasn't when I created. I thought of moving after I finished the work to make it all generic and redundant, then forgot!!!